runtime/compiler/aarch64/codegen/J9TreeEvaluator.cpp

```diff
@@ -6578,6 +6578,267 @@ static TR::Register* inlineIntrinsicIndexOf(TR::Node* node, TR::CodeGenerator* c
+   TR::Register *vtmp2Reg = cg->allocateRegister(TR_VRF);
+
+   TR::RegisterDependencyConditions *dependencies = new (cg->trHeapMemory()) TR::RegisterDependencyConditions(14, 14, cg->trMemory());
+   dependencies->addPreCondition(s1Reg, TR::RealRegister::NoReg);
+   dependencies->addPreCondition(s2Reg, TR::RealRegister::NoReg);
+   dependencies->addPreCondition(s2lenReg, TR::RealRegister::NoReg);
+   dependencies->addPreCondition(maxReg, TR::RealRegister::NoReg);
+   dependencies->addPreCondition(resultReg, TR::RealRegister::NoReg);
+   dependencies->addPreCondition(s1addrReg, TR::RealRegister::NoReg);
+   dependencies->addPreCondition(s1idxReg, TR::RealRegister::NoReg);
+   dependencies->addPreCondition(s2addrReg, TR::RealRegister::NoReg);
+   dependencies->addPreCondition(s2idxReg, TR::RealRegister::NoReg);
+   dependencies->addPreCondition(tmp1Reg, TR::RealRegister::NoReg);
+   dependencies->addPreCondition(tmp2Reg, TR::RealRegister::NoReg);
+   dependencies->addPreCondition(s2firstCharReg, TR::RealRegister::NoReg);
+   dependencies->addPreCondition(vtmp1Reg, TR::RealRegister::NoReg);
+   dependencies->addPreCondition(vtmp2Reg, TR::RealRegister::NoReg);
+
+   dependencies->addPostCondition(s1Reg, TR::RealRegister::NoReg);
+   dependencies->addPostCondition(s2Reg, TR::RealRegister::NoReg);
+   dependencies->addPostCondition(s2lenReg, TR::RealRegister::NoReg);
+   dependencies->addPostCondition(maxReg, TR::RealRegister::NoReg);
+   dependencies->addPostCondition(resultReg, TR::RealRegister::NoReg);
+   dependencies->addPostCondition(s1addrReg, TR::RealRegister::NoReg);
+   dependencies->addPostCondition(s1idxReg, TR::RealRegister::NoReg);
+   dependencies->addPostCondition(s2addrReg, TR::RealRegister::NoReg);
+   dependencies->addPostCondition(s2idxReg, TR::RealRegister::NoReg);
+   dependencies->addPostCondition(tmp1Reg, TR::RealRegister::NoReg);
+   dependencies->addPostCondition(tmp2Reg, TR::RealRegister::NoReg);
+   dependencies->addPostCondition(s2firstCharReg, TR::RealRegister::NoReg);
+   dependencies->addPostCondition(vtmp1Reg, TR::RealRegister::NoReg);
+   dependencies->addPostCondition(vtmp2Reg, TR::RealRegister::NoReg);
+
+   TR::LabelSymbol *startLabel = generateLabelSymbol(cg);
+   TR::LabelSymbol *outerLoopLabel = generateLabelSymbol(cg);
+   TR::LabelSymbol *firstCharLoopLabel = generateLabelSymbol(cg);
+   TR::LabelSymbol *firstCharMatchedLabel = generateLabelSymbol(cg);
+   TR::LabelSymbol *arrayCmpVectorLoopLabel = generateLabelSymbol(cg);
+   TR::LabelSymbol *arrayCmpByteLoopLabel = generateLabelSymbol(cg);
+   TR::LabelSymbol *unmatchedLabel = generateLabelSymbol(cg);
+   TR::LabelSymbol *notFoundLabel = generateLabelSymbol(cg);
+   TR::LabelSymbol *doneLabel = generateLabelSymbol(cg);
+
+   startLabel->setStartInternalControlFlow();
+   doneLabel->setEndInternalControlFlow();
+
+   generateLabelInstruction(cg, TR::InstOpCode::label, node, startLabel);
+
+   const int32_t vecWidth = 16;
+
+   // Adresses of array elements
```

Comment:
sp. Addresses

Reply:
Fixed the typo.  I made no changes in other lines.